api/openapi-spec/swagger.json

```diff
@@ -46070,35 +46070,6 @@
       "required": true
      }
     ]
-   },
-   "/version/": {
```

Comment:
Sorry if I still don't see the problem. `/version/openshift` and `/version/apiextensions-apiserver` can be registered the same way as `/version` in both cases. How gorestful hides those endpoints and the new implementation doesn't?